themes/default/public/styles/raneto.css

```diff
@@ -232,16 +232,77 @@
 
 .lang-social-bar {
   position: absolute;
-  right: 0;
-  bottom: 0;
+  right: 50px;
+  bottom: 50px;
+}
+
+.lang-social-bar a {
+  cursor: pointer;
+  color: #333;
+}
+
+.lang-social-bar .social-icon {
+  float: right;
+  font-size: 24px;
+  padding-left: 8px;
+}
+
+.lang-social-bar button {
+  background: #b3ba3f;
+  border: 0;
+  padding: 11px 15px 10px 15px;
+  border-radius: 32px;
+  color:  white;
+}
+
+.lang-social-bar button:hover {
+  background: #dae611;
 }
 
 .lang-french {
   display: none;
+  float: left;
+  padding-right: 2px;
+  padding-top: 1px;
+  font-size: 16px;
 }
 
 .lang-english {
   display: none;
+  float: right;
+  padding-right: 2px;
+  padding-top: 1px;
+  font-size: 16px;
+}
+
+.contact-donate-bar {
+  position: absolute;
+  top: 30px;
+  right: 30px;
+}
+
+.contact-donate-bar button {
+  background: #fff;
+  border-color: #b3ba3f;
+  border-width: 1px;
+  border-radius: 32px;
+  padding: 10px 20px 10px 20px;
```

Comment:
@willmero do we want pixels or ems? That seems to be the standard in the other files.